docs/Shapes.md

```diff
@@ -170,12 +170,18 @@ from fpdf import FPDF
 pdf = FPDF()
 pdf.add_page()
 pdf.set_fill_color(r=255, g=0, b=255)
-pdf.bezier([(20, 80), (40, 20), (60, 80)])
+pdf.bezier([(20, 80), (40, 20), (60, 80)], style="DF")
 pdf.output("bezier.pdf")
 ```
 
 ![](bezier.png)
 
+One of the nice properties of bezier curves is that they can be chained:
+
+![](bezier-chaining.png)
```

Comment:
Image file is missing.

btw.: If we already advertize chaining beziers, it might be helpful to at least give hint at how to do that correctly. (I assume for a smooth continuation the three control points around a join must line up).

Another unrelated nitpick: I think most graphics on this page could be significantly smaller. There's no real point in filling almost a full screen with a dashed line or a filled circle...

Reply:
> btw.: If we already advertize chaining beziers, it might be helpful to at least give hint at how to do that correctly. (I assume for a smooth continuation the three control points around a join must line up).

Good point.

> Another unrelated nitpick: I think most graphics on this page could be significantly smaller. There's no real point in filling almost a full screen with a dashed line or a filled circle...

Agreed!